src/views/Lottery/components/Hero.tsx

```diff
@@ -70,12 +94,17 @@ const Hero = () => {
     <StyledHero>
       <StyledContainer>
         <LeftWrapper>
-          <Title>{TranslateString(708, 'The CAKE Lottery')}</Title>
-          <Blurb>{TranslateString(710, 'Buy tickets with CAKE')}</Blurb>
-          <Blurb>{TranslateString(712, 'Win if 2, 3, or 4 of your ticket numbers match!')}</Blurb>
+          <Title>{TranslateString(999, 'The Lottery Is Changing!')}</Title>
+          <ComeBack>{TranslateString(999, 'Come back soon!')}</ComeBack>
+          <LearnMore>
```

Comment:
I think you could simplify this too and avoid the styled comp?

`<Text bold color="primary">`